README.md

```diff
@@ -137,152 +123,76 @@ Example command:
 
 ### Setup
 
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
+Clone this repository to your desired folder .
 
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
 
 ### Install
 
-Install this project with:
-
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
-```
---->
+No Installation needed
 
 ### Usage
 
-To run the project, execute the following command:
-
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
-
-### Deployment
-
-You can deploy this project using:
-
-<!--
-Example:
-
-```sh
+To run the project, just open (double click on) the file named A.html
 
-```
- -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
 
-👤 **Author1**
+👤 **Célestin SOUOP (Mzirkof)**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/mzirkof)
+- Twitter: [@twitterhandle](https://twitter.com/mzirkof)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/celestin-souop-58341a109/)
```

Comment:
[Optional] Great work on listing where people can find you, but it seems you forgot to replace the text between the brackets
`[@githubhandle](https://github.com/mzirkof)`

You could update them to something like:
`[mzirkof](https://github.com/mzirkof)`

Reply:
Ok